.github/workflows/ci.yml

```diff
@@ -17,4 +17,5 @@ jobs:
         run: |
           python3 -m pip install --upgrade pip     # install pip
           python3 -m pip install snakemake         # ...and snakemake
+          python3 -m pip install pandas         # ...and pandas
```

Comment:
no need for a change here, but note that you can `pip install snakemake pandas` all in one ;)